src/main/kotlin/hs/kr/equus/user/domain/user/mapper/UserMapper.kt

```diff
@@ -0,0 +1,18 @@
+package hs.kr.equus.user.domain.user.mapper
+
+import hs.kr.equus.user.domain.user.domain.UserJpaEntity
+import hs.kr.equus.user.domain.user.model.User
+import org.springframework.stereotype.Component
+
+@Component
+class UserMapper {
+    fun toDomain(user: UserJpaEntity): User {
+        return User(
+            user.id,
+            user.phoneNumber,
+            user.name,
+            user.isStudent,
+            user.receiptCode
+        )
+    }
```

Comment:
1. 코드 단순화를 위해 run 함수 쓰면 좋을 것 같아요
2. 인자 구분을 위해 인자명 명시하면 좋을 것 같아요
```suggestion
    fun toDomain(user: UserJpaEntity) = user.run {
        User(
            id = id,
            phoneNumber = phoneNumber,
            name = name,
            isStudent = isStudent,
            receiptCode = receiptCode
        )
    }
```